sdk/core/azure-core/inc/azure/core/http/policies/policy.hpp

```diff
@@ -173,6 +173,14 @@ namespace Azure { namespace Core { namespace Http { namespace Policies {
      */
     bool EnableCertificateRevocationListCheck{false};
 
+    /**
+     * @brief Verify peer's SSL certificate.
+     *
+     * @remark This field is only used if the customer has not specified a default transport
+     * adapter. If the customer has set a Transport adapter, this option is ignored.
+     */
+    bool SslVerifyPeer{true};
```

Comment:
In general, all of our options "enable" a behavior. So we have "EnableSystemDefaultProxy", "EnableCertificateRevocationListCheck", etc. This option is backwards because the default is true.

In general, I prefer options to be defaulted to false and when set to "true" it defines a new behavior.

So instead of `SslVerifyPeer`, I think something like `DisableTlsPeerValidation` or `DisableTlsCertificateValidation` is probably a better name for this option (the encryption technology previously (in 2000 or so) called SSL (Secure Sockets Layer) is now called "TLS" (Transport Layer Security)).

Also if you could add a comment indicating that disabling TLS security is generally a bad idea because it allows malicious actors to spoof the target server and should never be enabled in production code.